backend/services/stats_service.py

```diff
@@ -311,7 +335,14 @@ def get_user_contributions(project_id: int) -> ProjectContributionsDTO:
                     picture_url=r.picture_url,
                     mapped=r.mapped,
                     validated=r.validated,
-                    total=r.total,
+                    split=r.split,
+                    marked_bad_imagery=r.bad_imagery,
+                    invalidated=r.invalidated,
+                    total=r.mapped
+                    + r.validated
+                    + r.bad_imagery
+                    + r.invalidated
+                    + r.split,
```

Comment:
```suggestion
                    total=r.mapped + r.validated,
```
Remove the addition of bad_imagery, invalidated, and split tasks from the total count since this shows up on tasks validated and invalidated by a user in the frontend.
![split_task_counted](https://user-images.githubusercontent.com/67958673/214237965-b545c1b6-f604-4a4e-89f7-10621308cb4c.png)

Reply:
Implemented that.